dockertest/config.py

```diff
@@ -203,6 +203,7 @@ def merge_write(self, fileobject):
         scp = SafeConfigParser()
         # Safe if file doesn't exist
         scp.read(fileobject.name)
+        # N/B: This won't work with DEFAULTS
```

Comment:
I don't, I looked at it briefly but couldn't figure it out.  It may be something in the ``ConfigParser`` module.